sdk/core/ci.yml

```diff
@@ -24,7 +24,29 @@ resources:
       name: Azure/azure-sdk-tools
       endpoint: azure
 
-jobs:
-- template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
-  parameters:
-    ServiceDirectory: '*'
\ No newline at end of file
+stages:
```

Comment:
Is there any reason this is not part of the client template?

Reply:
To get the docs publishing per package, I needed a separate Doc job per package. I couldn't add the strategy matrix in the archetype-client template.

Reply:
I don't understand why you cannot loop over each of the packages and do the publishing for each? Even if that doesn't work we should probably do something like https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/ci.yml#L39 and list the artifacts and then let the template generate one per-package. We are going to need to do that soon for the unified pipeline work that @mitchdenny is working on, see https://github.com/Azure/azure-sdk-for-net/pull/8258 as an example.

Reply:
I will align this with the unified pipeline. I guess I can loop over the Artifact array @mitchdenny is using then run the docs for each.

Reply:
I have updated doc generation to work with unified pipelines. Once @mitchdenny 's changes are merged I will update all the ci's to work with this. I wonder whats the plan for management libraries,